tsconfig-es.json

```diff
@@ -0,0 +1,17 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
```

Comment:
Should be es6 or esnext so es modules are generated

Reply:
Yeah will fix that...

Reply:
fixed

Reply:
Thank you!